api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -109,6 +112,30 @@ void shouldReturnEligibleWhenNotDuplicateAndEligible() {
         verify(duplicateClaimChecker).liveClaimExistsForHousehold(IDENTITY_AND_ELIGIBILITY_RESPONSE);
     }
 
+    @Test
+    void shouldEvaluateClaimForGivenPaymentCycle() {
+        //Given
+        PaymentCycle paymentCycle = aValidPaymentCycle();
+        EligibilityAndEntitlementDecision decision = anEligibleDecision();
+        given(client.checkIdentityAndEligibility(any())).willReturn(IDENTITY_AND_ELIGIBILITY_RESPONSE);
+        given(paymentCycleEntitlementCalculator.calculateEntitlement(any(), any(), any(), any())).willReturn(VOUCHER_ENTITLEMENT);
+        given(eligibilityAndEntitlementDecisionFactory.buildDecision(any(), any(), any(), anyBoolean())).willReturn(decision);
+        Claimant claimant = paymentCycle.getClaim().getClaimant();
+        LocalDate cycleStartDate = LocalDate.now();
+
+        //When
+        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateClaimantForPaymentCycle(claimant, cycleStartDate, paymentCycle);
+
+        //Then
+        assertThat(result).isEqualTo(decision);
+        List<LocalDate> dobOfChildrenUnder4 = IDENTITY_AND_ELIGIBILITY_RESPONSE.getDobOfChildrenUnder4();
+        Optional<LocalDate> expectedDeliveryDate = Optional.of(claimant.getExpectedDeliveryDate());
```

Comment:
Might be clearer to use `EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS` here as its quite hard to figure out what the EDD should be otherwise.

Reply:
Done.